sympa.spec

```diff
@@ -8,7 +8,7 @@
 
 # Fonts
 #
-%global unbundle_fontawesome       1
+%global unbundle_fontawesome       0%{?fedora}
```

Comment:
As fontawesome has an higher version in EL6 (from EPEL) than in EL7 (from base OS), it's still fine to unbundle it in EL6 so %{?el6} needs to be added.
Also, it's a bit sad to completely remove the unbundling in EL7 for just one icon. Maybe the we can either patch the EL build to use another icon on EL7 ? Or do it unconditionally in upstream sympa ?

Reply:
FYI.

These are not included in 4.1 but used by sympa:
  - [fa-trash](https://fontawesome.com/v4.7.0/icon/trash)
  - [fa-user-plus](https://fontawesome.com/v4.7.0/icon/user-plus)
  - [fa-user-times](https://fontawesome.com/v4.7.0/icon/user-times)

On other distributors:
  - Sympa source tarball bundles 4.3.
  - Debian (stretch) provides 4.7.
  - FreeBSD provides 5.3.

Reply:
ok, fine by me to not unbundle for EL7 then. I'd still like to have EL6 unbundle though.